### ISSUE TITLE
Write statistics

### ISSUE THREAD

**Issue by jonashaag** [pacman82/odbc2parquet#486]
It seems like no Parquet column statistics (like min/max value) are written by this tool.

Would be nice to write add an option to add statistics or even enable them by default.

**Comment by pacman82** [comment]
I assumed the `parquet-rs` crate would do so by default. Seems I am wrong. What is a good way to test statistics have been written?

**Comment by pacman82** [comment]
Note to self: <https://docs.rs/parquet/latest/parquet/file/properties/struct.WriterProperties.html#method.statistics_enabled>

Happy to change the default, still wonder how to test this, though.

**Comment by pacman82** [comment]
Extracting statistics from a file written with `odbc2parquet` worked for me. To do this I used a piece of Python code kindly supplied by Uwe on Stack Overflow:

```python
import pyarrow.parquet as pq

pq_file = pq.ParquetFile("tmp.par")
# Get metadata for the i-th RowGroup
rg_meta = pq_file.metadata.row_group(0)
# Get the "max" statistic for the k-th column
max_of_col = rg_meta.column(0).statistics.max
print(max_of_col)
```

Now, I wonder, what makes you think `odbc2parquet` does not write statistics? Are the statistics I saw calculated on demand?

**Comment by jonashaag** [comment]
Interestingly this seems to be the case with a single specific table only:


<img width="881" alt="image" src="https://github.com/pacman82/odbc2parquet/assets/175722/3a3b4797-d1e1-409a-b1be-86600f227576">

I'll try to create a reproducer next week.

**Comment by jonashaag** [comment]
Minimal reproducer on my SQL Server instance:

```
CREATE TABLE tmp (x CHAR(1))
INSERT INTO tmp (x) VALUES ('a')
```

Fetch with `SELECT * FROM tmp`.

<img width="550" alt="Screenshot 2023-11-19 at 11 07 20" src="https://github.com/pacman82/odbc2parquet/assets/175722/e926efc1-c838-44ea-b024-1daef8e759d2">

**Comment by pacman82** [comment]
I can reproduce this, but I wonder if this is an issue. It `parquet-rs` seems to generate statistics for e.g. integer columns, but not for text. Explicitly enabling writing statistics in the writer properties does not change it. So if this is an issue, I think I would need to send it upstream. I wonder however if I am missing another flag I could set or if this could even be considered sensible behavior for text columns.

**Comment by pacman82** [comment]
Am I also right in assuming it is specifically about the min/max statistic?

**Comment by pacman82** [comment]
Feel free to link this issue to the upstream ticket.

**Comment by jonashaag** [comment]
The ticket was unrelated. My ticket: https://github.com/apache/arrow-rs/issues/5270

**Comment by pacman82** [comment]
This behavior is fixed in `odbc2parquet 4.1.2` as it uses parquet 50.
